src/Domain/DomainHelpers.hpp

```diff
@@ -81,3 +81,13 @@ wedge_coordinate_maps(double inner_radius, double outer_radius,
                       bool use_equiangular_map,
                       double x_coord_of_shell_center = 0.0,
                       bool use_half_wedges = false) noexcept;
+
+/// \ingroup ComputationalDomainGroup
+/// These are the ten Frustums used in the BBH DomainCreator. If the cubes
+/// enveloping the two Shells each have a side length of 2L, then the argument
+/// that should be passed to `frustum_parameter_l` should be L. The ten frustums
+/// make up a cube of their own, of side length 2 * `frustum_parameter_t`.
```

Comment:
half_length_outer_cube and half_length_inner_cubes (and if easy, get rid of the halves)